chanzuckerberg/idseq-web
# [Report Page] Initial table view

# Description

* Implements a table view based on react virtualized.
* Currently only for development purposes and guarded by flag.
* Current status: report page only, no corner cases handled (genus shows up blank)

# Tests

* Open show_v2 route on a samples /samples/<smaple_id>/show_v2
* See report table
* Test sorting of all columns (asc and desc)
* Test collapsing genus and species
